package.json

```diff
@@ -16,5 +16,8 @@
     "webpack": "^4.39.3",
     "webpack-cli": "^3.3.7",
     "webpack-merge": "^4.2.2"
+  },
+  "dependencies": {
+    "ramda": "^0.26.1"
```

Comment:
we need reselect instead of this (maybe this too, but not for props fn)

Reply:
I can drop `ramda` in case with [selector function](https://github.com/one-more/micro-frontends/pull/16#discussion_r329363719)

Reply:
right